sdk/identity/identity/src/util/processUtils.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import internalCommandExists from "command-exists";
```

Comment:
I'm not against this, but it seems like we could just try to exec `pwsh -v` or `powershell /?` to test if the user has powershell instead of taking a dependency.

Reply:
I'll do this change! Thanks!

Reply:
+1 Since this seems like an easy dep to get hacked by